src/valhalla_benchmark_loki.cc

```diff
@@ -119,37 +124,53 @@ bool ParseArguments(int argc, char* argv[]) {
   } catch (std::exception& e) {
     std::cerr << "Unable to parse command line options because: " << e.what() << "\n"
               << "This is a bug, please report it at " PACKAGE_BUGREPORT << "\n";
-    return false;
+    return 1;
   }
 
   if (vm.count("help")) {
     std::cout << options << "\n";
-    return true;
+    return -1;
   }
 
   if (vm.count("version")) {
     std::cout << "loki_benchmark " << VALHALLA_VERSION << "\n";
-    return true;
+    return 0;
```

Comment:
Should it return `-1` instead, so that it only prints the version and stops further execution of `main`?

Reply:
good eyes! this one is like `help` as well